source/artemisd/utils/ext.d

```diff
@@ -13,7 +13,7 @@ bool isEmpty(BitArray ba)
     return true;
 }
 
-long nextSetBit(BitArray ba, size_t index)
+size_t nextSetBit(BitArray ba, size_t index)
```

Comment:
How does this work? `size_t` is unsigned and function often returns negative value.